cmd/internal/cli/push.go

```diff
@@ -32,6 +33,24 @@ var PushCmd = &cobra.Command{
 	Args:                  cobra.ExactArgs(2),
 	PreRun:                sylabsToken,
 	Run: func(cmd *cobra.Command, args []string) {
+		// if we can load config and if default endpoint is set, use that
```

Comment:
Not sure how to fix this, but it really looks like there's a lot of code that's duplicated here in a few spots, is it possible for us to de-duplicate somehow?